src/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,160 @@
+        >
+          <span className="sr-only">Previous month</span>
+          <ChevronLeftIcon className="w-5 h-5" aria-hidden="true" />
+        </button>
+        <h2 className="flex-auto font-semibold text-center text-cu-black-900">
+          {format(firstDayCurrentMonth, 'MMMM yyyy')}
+        </h2>
+        <button
+          onClick={nextMonth}
+          type="button"
+          className={`${styles.prevNextArrows}`}
+        >
+          <span className="sr-only">Next month</span>
+          <ChevronRightIcon className="w-5 h-5" aria-hidden="true" />
+        </button>
+      </div>
+
+      <div className={`${styles.calendarGrid} text-xs text-cu-black-600`}>
+        <div>S</div>
+        <div>M</div>
+        <div>T</div>
+        <div>W</div>
+        <div>T</div>
+        <div>F</div>
+        <div>S</div>
+      </div>
+
+      <div
+        className={`${styles.calendarGrid} text-sm rounded-lg bg-cu-black-50 isolate border border-cu-black-100 overflow-hidden`}
+      >
+        {days.map((day, dayIdx) => (
+          <div
+            key={day.toString()}
+            className={classNames(
+              dayIdx === 0 && colStartClasses[getDay(day)],
+              'py-2 bg-white'
+            )}
+          >
+            <button
+              type="button"
+              disabled={isBefore(day, today)}
+              onClick={() => setSelectedDay(day)}
+              className={classNames(
+                isEqual(day, selectedDay) && 'text-white',
+                !isEqual(day, selectedDay) &&
+                  isToday(day) &&
+                  'text-cu-red hover:text-cu-black-800',
+                !isEqual(day, selectedDay) &&
+                  !isToday(day) &&
+                  isSameMonth(day, firstDayCurrentMonth) &&
+                  'text-cu-black-900',
+                !isEqual(day, selectedDay) &&
+                  !isToday(day) &&
+                  !isSameMonth(day, firstDayCurrentMonth) &&
+                  'text-cu-black-400',
+                isEqual(day, selectedDay) && isToday(day) && 'bg-cu-red',
+                isEqual(day, selectedDay) && !isToday(day) && 'bg-cu-red',
+                !isEqual(day, selectedDay) &&
+                  'hover:bg-cu-red hover:text-white',
+                (isEqual(day, selectedDay) || isToday(day)) && 'font-semibold',
+                'mx-auto flex h-8 w-8 items-center justify-center rounded-full disabled:bg-cu-black-50'
+              )}
+            >
+              <time dateTime={format(day, 'yyyy-MM-dd')}>
+                {format(day, 'd')}
+              </time>
+            </button>
+
+            <div className="w-1 h-1 mx-auto mt-1">
+              {events.some(event =>
+                isSameDay(parseISO(event.startDatetime), day)
+              ) && <div className="w-1 h-1 rounded-full bg-sky-500"></div>}
+            </div>
+          </div>
+        ))}
+      </div>
+      <h2 className="mt-5 font-semibold text-center text-cu-black-800">
```

Comment:
Don't think we want this in the final component, maybe this could move to an onClick that gets posted in the action tab of Storybook. See button component as this does something similar.

Reply:
Thank you Troy, that helps a lot, I was think to remove the display selected date info, those are for test only, to make sure the date is correctly selected. In event cal, it will cooperate with list event to display the corresponding event(s).